src/libaudqt/info-widget.cc

```diff
@@ -116,6 +127,34 @@ EXPORT bool InfoWidget::updateFile ()
     return m_model->updateFile ();
 }
 
+void InfoWidget::toClipboard ()
+{
+    QAction *action = qobject_cast<QAction*>(sender ());
```

Comment:
Qt recommends accessing the sender of an event, with good reason.  The cleaner way to do this would be to capture the QString we want to copy in a lambda function, and connect the signal to the lambda.  (This is one of the coolest features in Qt5, in my opinion.)